R/moma_arguments.R

```diff
@@ -1,80 +1,83 @@
 #' Sparsity-inducing penalty in \code{MoMA}
+#'
 #' In the package \code{MoMA}, we support the following sparsity-inducing
 #' penalty functions.
 #' \itemize{
-#'   \item{\code{\link{lasso}}} TODO
-#'   \item{\code{\link{mcp}}} TODO
-#'   \item{\code{\link{scad}}} TODO
-#'   \item{\code{\link{slope}}} TODO
-#'   \item{\code{\link{grplasso}}} TODO
-#'   \item{\code{\link{fusedlasso}}} TODO
-#'   \item{\code{\link{l1tf}}} TODO
-#'   \item{\code{\link{spfusedlasso}}} TODO
-#'   \item{\code{\link{cluster}}} TODO
+#'   \item{\code{\link{moma_lasso}}}: sparsity
+#'   \item{\code{\link{moma_mcp}}}: adaptive sparsity
+#'   \item{\code{\link{moma_scad}}}: adaptive sparsity
+#'   \item{\code{\link{moma_slope}}}: adaptive sparsity
+#'   \item{\code{\link{moma_grplasso}}}: group-wise sparsity
+#'   \item{\code{\link{moma_fusedlasso}}}: piece-wise constant
+#'   \item{\code{\link{moma_spfusedlasso}}}: sparsity and piece-wise constant
+#'   \item{\code{\link{moma_l1tf}}}: piece-wise quadratic
+#'   \item{\code{\link{moma_cluster}}}: structured sparsity
 #' }
-#' @name moma_sparsity
+#' These functions specify the value of the \code{u_sparse,v_sparse} arguments in the
+#'  \code{moma_*pca} series of functions, and the \code{x_sparse,y_sparse} arguments
+#' in the \code{moma_*cca} and \code{moma_*lda} series of functions.
+#'
+#' All functions
+#' above share two common parameters: \code{lambda} and \code{select_scheme}, which are
+#' described in the Arguments section below.
+#' @name moma_sparsity_options
+#' @param ... Force users to specify arguments by names.
+#' @param lambda A vector containing penalty values
+#' @param select_scheme A char being either "b" (nested BIC search) or "g" (grid search).
+#'
+#' Selection of tuning parameters (four parameters in our case, \code{alpha_u}, \code{alpha_v}, \code{lambda_u}, \code{lambda_v}) is a problem in MoMA. Currently we provide
```

Comment:
Let's not call this a problem - too negative. 

Maybe something more like: 

MoMA provides a flexible framework for regularized multivariate analysis with several tuning parameters for different forms of regularization. To assist the user in selecting these parameters ....

Reply:
Fixed.